experiment/measurer/coverage_printer_utils.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility DataFrameContainer class and functions for report generation on
+segment and function coverage for the entire experiment
+(all Fuzzer-benchmark-trial combinations)."""
+
+import os
+import hashlib
+import pandas as pd
+
+from common import experiment_path as exp_path
+from common import filestore_utils
+from experiment.measurer import coverage_utils
+
+
+class DataFrameContainer:  # pylint: disable=too-many-instance-attributes
```

Comment:
I think this name is very vague and would like a better one.
Maybe CoverageOverTimeData or DetailedCoverageData?